ui/hooks/gasFeeInput/test-utils.js

```diff
@@ -122,7 +121,7 @@ export const generateUseSelectorRouter =
         },
       };
     }
-    if (selector === getSelectedAccount) {
+    if (selector.toString().includes('getTargetAccount')) {
```

Comment:
Hack because the selector is actually a lambda, to allow `getTargetAccount` to accept a second parameter.  Therefore it can't use function equality like the others.

A better approach to this kind of mocking, would keep the selectors from the product code and mock the state instead.  Better to mock the lowest level thing, so that the product selectors are tested too if they change.